### PR TITLE
Add TokensPlus Placeholder

### PR DESCRIPTION
**Pull request by CMPSCjg**
### Type

- [ ] Internal change (Doesn't affect end-user).
- [ ] External change (Does affect end-user).
- [x] Wiki (Changes towards the [Wiki]).
- [ ] Other: __________ 

### Description

Updates the placeholder wiki README to include TokensPlus and the placeholder defined internally.

Closes N/A 

[Wiki]: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki

### CODE REVIEW

File: wiki/Placeholders.md [hunk 1]
```diff
@@ -4404,6 +4405,13 @@ Replace `XXX` with the enchantment.
 ```
 ----
 
+- ### **[TokensPlus](https://www.spigotmc.org/resources/tokensplus.90507/)**
+> NO DOWNLOAD COMMAND
```
Comment (Andre601) [inline wiki/Placeholders.md:4409]: Add an empty line here to not have the code block as quote.

File: wiki/Placeholders.md [hunk 2]
```diff
@@ -4404,6 +4405,13 @@ Replace `XXX` with the enchantment.
 ```
 ----
 
+- ### **[TokensPlus](https://www.spigotmc.org/resources/tokensplus.90507/)**
```
Comment (Andre601) [inline wiki/Placeholders.md:4408]:
```suggestion
- ### **[TokensPlus](https://www.spigotmc.org/resources/90507/)**
```

**Review by Andre601 — CHANGES_REQUESTED** [review]
Two minor changes to make.

Also, remember to also add your plugin to the page [Plugins using PlaceholderAPI](https://github.com/PlaceholderAPI/PlaceholderAPI/blob/docs/wiki/wiki/README.md#plugins-using-placeholderapi)

File: wiki/Placeholders.md (hunk 1 above)
Comment (CMPSCjg) [reply wiki/Placeholders.md:4409]: Added empty line below >NO DOWNLOAD COMMAND

**Review by Andre601 — APPROVED** [review]

**Pull request merged**